app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -421,6 +421,11 @@
             </div>
             <div class="content">
               <div class="header">{{data.event.tax.name}}</div>
+              {{#if data.event.tax.shouldSendInvoice}}
+                <div class="description">
+                  {{t 'Company Name '}}{{data.event.tax.registeredCompany}}
```

Comment:
A simple `{{data.event.tax.registeredCompany}}` would do. (I think we should follow the existing design of card and not introduce statics like 'Company Name'

Reply:
@uds5501 That could be somewhat confusing than , As we will have `Tax Name` as well as `Company Name` , Thats why added a static

Reply:
@kushthedude Okay, fair point. Please address the above issue then.